ansible/roles/zsh/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+- name: Install zsh
+  ansible.builtin.apt:
+    name: zsh
+    state: present
+  when: ansible_facts['os_family'] == 'Debian'
+- name: Set up zsh config
+  template:
+    src: zshrc
+    dest: "~{{ item }}/.zshrc"
+    mode: u+rw-x,o-rwx,g-rwx
+    owner: "{{ item }}"
+    group: "{{ item }}"
+  loop: "{{ users }}"
```

Comment:
if users is a role variable, it should have a default.  `users` is also super generic, so we should prefix it.  Maybe `zsh_users` at least?

Reply:
What's the right convention here? I was expecting that the variable would be explicitly passed into the role (hence a contextual name is ok) but perhaps in more of a mixed-mode setting (so not the Kairos nodes but more of a pet?)

Reply:
I've not seen a consistent convention, but the one I like the most is where they do something like {role}__{variable}}.  Lots of projects do a single underscore too, but I like the double because it makes it easier to see that it's meant to be a namespace.

Having it be a bit more unique like this let's someone set it for all hosts too so they don't need to include it in their play.  This is something I do for somethings.